docs/team/tanhl2000.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Tan Hong Liang's Project Portfolio Page
+---
+
+### Project: Duke The Market
+
+Duke The Market is a desktop contact management system application used for managing customer contacts in department stores. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **New Feature**: 
+  * To be added soon
+
+* **New Feature**: To be added soon
+
+* **Code contributed**: To be added soon
+
+* **Project management**:
+  * To be added soon
+
+* **Enhancements to existing features**:
+  * To be added soon
+
+* **Documentation**:
+  * To be added soon
+
+* **Community**:
+  * To be added soon
+
+* **Tools**:
+  * To be added soon
```

Comment:
From the checks: https://github.com/AY2223S1-CS2103-F09-2/tp/pull/32/checks,
seems like the codecov is failing due as there's no newline after line 32.